demo/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2020",
```

Comment:
Why did we change this?

Reply:
I was having some complaints from typescript in the demo, trying to inline BigInt variables like this: `0n`. We were targeting older browsers on the demo app, so I dropped those requirements by matching our tsconfig for the kit.